src/hotspot/share/gc/g1/g1Policy.cpp

```diff
@@ -780,6 +783,9 @@ void G1Policy::record_collection_pause_end(double pause_time_ms, bool concurrent
 
   update_rs_length_prediction();
 
+  // Is this the right place? Should it be in the below?
```

Comment:
Can a reviewer confirm this so that I can remove the comment?

Reply:
The comment can be removed imho.